activerecord/lib/active_record/schema_migration.rb

```diff
@@ -31,8 +35,10 @@ def create_table(limit=nil)
 
           connection.create_table(table_name, id: false) do |t|
             t.column :version, :string, version_options
+            t.column :created_at, :datetime, null: false
```

Comment:
instead of using a regular migration for a framework table, or the `rails update` task which won't run in production... I thought the framework should insure it is present itself on migrate.


if you run your first db:migrate and the table is not present, it gets created...
well if its there and the column is missing, add it.
I did this in my attempt to implement the exact same feature.

see link in my previous comment